cmd/dosa/main.go

```diff
@@ -38,7 +38,7 @@ type GlobalOptions struct {
 	Host        string   `long:"host" default:"127.0.0.1" description:"The hostname or IP for the gateway."`
 	Port        string   `short:"p" long:"port" default:"21300" description:"The hostname or IP for the gateway."`
 	Transport   string   `long:"transport" default:"tchannel" description:"TCP Transport to use. Options: http, tchannel."`
-	ServiceName string   `long:"service" default:"dosa-gateway" description:"The TChannel service name for the gateway."`
+	ServiceName string   `long:"service" description:"The TChannel service name for the gateway."`
```

Comment:
nit: we lost the default value information (I realize it is no longer a constant)